docs/_config.yml

```diff
@@ -37,30 +37,31 @@ kramdown:
 
 exclude:
   - jekyllbook
+  - en/index.md
```

Comment:
This is a hack since the default language has a special index.md. We symlink it and exclude en/index.md.